library/lua/gui/widgets.lua

```diff
@@ -399,16 +412,33 @@ function Label:onRenderBody(dc)
     render_text(self,dc,0,0,text_pen,self.text_dpen,is_disabled(self))
 end
 
+function Label:scroll(nlines)
+    local n = self.start_line_num + nlines
+    n = math.min(n, self:getTextHeight() - self.frame_body.height)
```

Comment:
Looks like this needs a `+1` at the end - I can address

Reply:
Ah, I was letting it overlap a line so the reader wouldn't be unsure if they were missing anything

Reply:
To clarify, I was seeing a cut-off line at the end ("modifications!" can't be viewed):

![image](https://user-images.githubusercontent.com/3719547/106372489-36983900-633e-11eb-80d9-7205d1e53eee.png)


which this fixed:
```suggestion
    n = math.min(n, self:getTextHeight() - self.frame_body.height + 1)
```